scripts/jest/matchers/toWarnDev.js

```diff
@@ -86,7 +86,8 @@ const createMatcherFor = (consoleMethod, matcherName) =>
         // doesn't match the number of arguments.
         // We'll fail the test if it happens.
         let argIndex = 0;
-        format.replace(/%s/g, () => argIndex++);
+        // console.* could have been called with a non-string e.g. `console.error(new Error())`
```

Comment:
Stray log

Reply:
I don't follow.

This comment supposed to explain why we need a string cast here. We used to assume `console.error` was always called with a string as its first argument. But when I migrated the test in this PR, it crashed at this line because `format` was not actually a string.